include/xkbcommon/xkbcommon-names.h

```diff
@@ -34,9 +34,13 @@
 #define XKB_MOD_NAME_SHIFT      "Shift"
 #define XKB_MOD_NAME_CAPS       "Lock"
 #define XKB_MOD_NAME_CTRL       "Control"
-#define XKB_MOD_NAME_ALT        "Mod1"
-#define XKB_MOD_NAME_NUM        "Mod2"
-#define XKB_MOD_NAME_LOGO       "Mod4"
+#define XKB_MOD_NAME_ALT        "Alt"
+#define XKB_MOD_NAME_META       "Meta"
+#define XKB_MOD_NAME_LOGO       "Super"
+#define XKB_MOD_NAME_GUI        "Super"
```

Comment:
Where does the GUI name come from? I don't think I've ever seen that one before

Reply:
I thought I’ve seen it in Wikipedia or W3C’s [“UI Events KeyboardEvent key Values”](https://www.w3.org/TR/uievents-key/#keys-modifier), but no 😅. There is also “OS”, but it is not much better than “Logo”.

I like “GUI” because this modifier is usually used for GUI shortcuts. But this may not always be the case, so let’s stick to previous and explicit modifier names.